native/packages/react-native-bpk-component-text-input/stories.js

```diff
@@ -62,57 +72,69 @@ class StatefulBpkTextInput extends Component<
-      initialValue="Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor." // eslint-disable-line max-len
-      multiline
-      style={styles.input}
-      autoGrow
-    />
-    <StatefulBpkTextInput
-      label="Valid input"
-      initialValue="Edinburgh"
-      valid
-      style={styles.input}
-    />
-    <StatefulBpkTextInput
-      label="Invalid input"
-      initialValue="Edinbvrgh"
-      valid={false}
-      validationMessage="'Edinbvrgh' is not a valid city."
-      style={styles.input}
-    />
-    <StatefulBpkTextInput
-      label="Non-editable input"
-      initialValue=""
-      editable={false}
-      style={styles.input}
-    />
-    <StatefulBpkTextInput
-      label="Password"
-      initialValue="letmein"
-      secureTextEntry
-      style={styles.input}
-    />
-    <StatefulBpkTextInput
-      label="Phone number"
-      initialValue="+441234567890"
-      keyboardType="phone-pad"
-      style={styles.input}
-    />
-  </ScrollView>
-));
+storiesOf('BpkTextInput', module)
+  .add('docs:text-inputs', () => (
+    <ScrollView>
+      <StatefulBpkTextInput
+        label="Input"
+        initialValue=""
+        style={styles.input}
+        placeholder="3 letter airport code"
+      />
+      <StatefulBpkTextInput
+        label="Input with value and description"
+        initialValue="Edinburgh"
+        description="Enter your destination."
+        style={styles.input}
+      />
+      <StatefulBpkTextInput
+        label="Input with multiline value"
+        initialValue="Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor." // eslint-disable-line max-len
+        multiline
+        style={styles.input}
+        autoGrow
+      />
+      <StatefulBpkTextInput
+        label="Valid input"
+        initialValue="Edinburgh"
+        valid
+        style={styles.input}
+      />
+      <StatefulBpkTextInput
+        label="Invalid input"
+        initialValue="Edinbvrgh"
+        valid={false}
+        validationMessage="'Edinbvrgh' is not a valid city."
+        style={styles.input}
+      />
+      <StatefulBpkTextInput
+        label="Non-editable input"
+        initialValue=""
+        editable={false}
+        style={styles.input}
+      />
+      <StatefulBpkTextInput
+        label="Password"
+        initialValue="letmein"
+        secureTextEntry
+        style={styles.input}
+      />
+      <StatefulBpkTextInput
+        label="Phone number"
+        initialValue="+441234567890"
+        keyboardType="phone-pad"
+        style={styles.input}
+      />
+    </ScrollView>
+  ))
+  .add('docs:text-inputs-with-accessory-view', () => (
+    <ScrollView>
+      <StatefulBpkTextInput
+        label="Phone number"
+        initialValue="+441234567890"
+        keyboardType="phone-pad"
+        style={styles.input}
+        accessoryView={<View style={styles.accessoryView} />}
+      />
+    </ScrollView>
```

Comment:
Can we add an example with a placeholder and no initial value?

e.g.
```
.add('docs:text-inputs-with-accessory-view', () => (
    <ScrollView>
      <StatefulBpkTextInput
        label="Phone number"
        initialValue="1234567890"
        keyboardType="phone-pad"
        style={styles.input}
        accessoryView={<View style={styles.accessoryView} />}
      />
      <StatefulBpkTextInput
        label="Phone number"
        initialValue=""
        keyboardType="phone-pad"
        style={styles.input}
        placeholder="E.g. 1234567890"
        accessoryView={<View style={styles.accessoryView} />}
      />
    </ScrollView>
  ));
```

Reply:
Yeah sure

Reply:
Actually can we not add all the other examples too? error, success etc?